### ISSUE TITLE
Entering a date manually on the Add Encounter Modal window throws an error 

### ISSUE THREAD

**Issue by richardmoizeau** [openmsupply/open-msupply#1183]
In Patients > Patient Detailed View > Add Encounter Modal: 

When entering a date manually for `Visit Date`, it automatically goes to the "Oops! Something's gone wrong." window: 

![Feb-14-2023 16-50-38](https://user-images.githubusercontent.com/74992958/218634937-648a5c5d-437a-407c-b54b-8ac6c72e08fc.gif)

Works fine when using the date picker. 

Demo: https://demo-open.msupply.org:88/dispensary/patients/8d2d45fa-b446-47fc-9629-5571a50feba3
version: 1.1.1

**Comment by roxy-dao** [comment]
Can't seem to reproduce this 🤔

**Comment by CarlosNZ** [comment]
Similarly for Start Time and End Time.

**Comment by CarlosNZ** [comment]
I can get started on this properly once #1262 is merged, as I'll use that framework to turn the modal into a JSON form.

**Comment by CarlosNZ** [comment]
> I can get started on this properly once #1262 is merged, as I'll use that framework to turn the modal into a JSON form.

We'll still need to fix this for standard (non-JSON FOrms) date/time entries (they all have the same problem), so will do that in this issue and have made a new issue for the more general change to use JSON Forms for the Add Encounter Modal -- #1264

**Comment by CarlosNZ** [comment]
Fixed by #1279
